pmlc/dialect/pxa/transforms/fusion.cc

```diff
@@ -189,6 +210,14 @@ struct FusionInfo {
       return false;
     }
 
+    // Add tiling if needed
+    if (needsTiling) {
+      if (reverseFusion)
+        performTiling(aInfo.op, tileSizes);
```

Comment:
I presume that the code for untiling isn't yet written?  I realize this is a WIP.

Reply:
yes, please see the latest checkin which has the tiling\untiling implemented